dashboard/app/controllers/api/v1/sections_controller.rb

```diff
@@ -45,13 +45,20 @@ def create
       return head :forbidden unless Section.can_be_assigned_course?(participant_audience, params[:participant_type])
     end
 
+    # If the new grades field exists, it takes precedence.
```

Comment:
i ended up having to modify the sections api to support `grades` and `grade` as keys, because as soon as I updated the JS to use `grades` it started sending a `grades` array to this api. because it's now using the new format, I gave that precedence here, but lmk if we think `grade` should take precedence instead.